inc/ruleimportcomputer.class.php

```diff
@@ -364,20 +370,15 @@ function findWithGlobalCriteria($input) {
                                     'sql_from'     => $sql_from];
                $sql_results = Plugin::doOneHook($plugin, "ruleImportComputer_getSqlRestriction",
                                                 $params);
-               $sql_where   = $sql_results['sql_where'];
-               $sql_from    = $sql_results['sql_from'];
+               $it_criteria = array_merge_recursive($it_criteria, $sql_results);
             }
          }
       }
 
-      $sql_glpi = "SELECT `glpi_computers`.`id`
-                   FROM $sql_from
-                   WHERE $sql_where
-                   ORDER BY `glpi_computers`.`is_deleted` ASC";
-      $result_glpi = $DB->query($sql_glpi);
+      $result_glpi = $DB->request($it_criteria);
```

Comment:
maybe i missed something, but i didn't found anywhere a line where you fill the `FROM` part

Reply:
It's filled from the plugin, as it was previously